test/crawler_web/controllers/user_registration_controller_test.exs

```diff
@@ -1,6 +1,8 @@
 defmodule CrawlerWeb.UserRegistrationControllerTest do
   use CrawlerWeb.ConnCase, async: true
 
+  @home_path "/home"
```

Comment:
Use the `Router helper` instead of hard code the path like this, maybe it's `Routes.home_path(Endpoint, :index)`

Reply:
IMO, we couldn't use `Routes.home_path(Endpoint, :index)` when asserting, we would need to update the home path whenever the code change, instead of the unit test always succeeding despite the path being changed. What do you think?

Reply:
Yup make sense @vnntsu but in the normal project, we are rarely change the routes :D anw, in Phoenix 1.7, there won't be Route Helper, instead we use the `path` manually everywhere in the codebase 

https://www.phoenixframework.org/blog/phoenix-1.7-released, the  Verified Routes part